src/apps/containers/components/ContainerSearch.tsx

```diff
@@ -100,7 +100,7 @@ const ContainerSearch: React.FC<Props> = ({
           />
         }
         label={
-          showOnlyVerified ? "Showing verified only" : "Showing all results"
+          showOnlyVerified ? `Showing verified ${title} only` : `Show Verified ${title} Only`
```

Comment:
I think it's best to keep the capitalisation consistent. Otherwise, LGTM.

```suggestion
          showOnlyVerified ? `Showing verified ${title} only` : `Show verified ${title} only`
```